libbeat/docs/newdashboards.asciidoc

```diff
@@ -18,74 +21,210 @@ export the Kibana dashboards.
+package containing the `import_dashboards` script. For example, to import the Metricbeat dashboards together with visualizations, 
+searches and the Metricbeat index pattern:
+
+[source,shell]
+-----------------
+$ scripts/import_dashboards -beat metricbeat
+-----------------
+
 The command has the following options:
 
 [source,shell]
 ----------------------------------------------------------------------
-$ ./import_dashboards.sh -h
+Usage: ./import_dashboards [options]
 
+Kibana dashboards are stored in a special index in Elasticsearch together with the searches, visualizations, and indexes that they use.
 
-Import the dashboards, visualizations and index patterns into Kibana.
+You can import the dashboards, visualizations, searches, and the index pattern for any Beat:
+  1. from a local directory:
+       	./import_dashboards -dir etc/kibana
+  2. from a local zip archive:
+       	./import_dashboards -beat metricbeat -file beats-dashboards-5.0.0-alpha6.zip
+  3. from a zip archive available online:
+       	./import_dashboards -beat metricbeat -url http://download.elastic.co/beats/dashboards/beats-dashboards-5.0.0-alpha6.zip
 
-The Kibana dashboards together with its dependencies are saved into a
-special index pattern in Elasticsearch (by default .kibana), so you need to
-specify the Elasticsearch URL and optionally a username and password.
+If the archive contains dashboards for multiple Beats, you need to pass the Beat name in -beat option:
+       	./import_dashboards -beat metricbeat -url http://download.elastic.co/beats/dashboards/beats-dashboards-%!s(MISSING).zip
 
-Usage:
-  import_dashboards.sh -url http://localhost:9200 -user admin:secret -index .kibana
+To import only the index-pattern for a single Beat, from the same version use:
+       	./import_dashboards -only-index -beat metricbeat
 
 Options:
-  -h | -help
-    Print the help menu.
-  -d | -dir
-    Local directory where the dashboards, visualizations, searches and index 
-    pattern are saved.
-  -l | -url
-    Elasticseacrh URL. By default is http://localhost:9200.
-  -u | -user
-    Username and password for authenticating to Elasticsearch using Basic
-    Authentication. The username and password should be separated by a
-    colon (i.e. admin:secret). By default no username and password are
-    used.
-  -i | -index
-    Kibana index pattern where to save the dashboards, visualizations,
-    index patterns. By default is .kibana.
-
+  -beat string
+       	Specify the Beat name, in case the archive contains the dashboards for multiple Beats.
+  -dir string
+       	Directory containing the subdirectories: dashboard, visualization, search, index-pattern. eg. kibana/
+  -es string
+       	Elasticsearch URL (default "http://127.0.0.1:9200")
+  -file string
+       	Zip archive file containing the Beats dashboards.
+  -i string
+       	Overwrites the Elasticsearch index name. For example you can replaces metricbeat-* with custombeat-*
```

Comment:
Typo. Change "you can replaces metricbeat..." to "you can replace metricbeat..."